docs/current_docs/api/arguments.mdx

```diff
@@ -226,6 +247,15 @@ The imported `float` type is a `number` underneath, so you can use it as you wou
 ```
 </TabItem>
 
+<TabItem value="Java">
+:::note
+You can either use the primitive `float` type or the boxed `java.lang.Float` type.
+:::
+
+```java file=./snippets/functions/arguments-return-values-float/java/src/main/java/io/dagger/modules/mymodule/MyModule.java
```

Comment:
Error https://dagger.cloud/dagger/traces/15eaad8737d00a3bda6529c50445287b

Reply:
Fixed in #9647